### PR TITLE
Update new-staff-performance-review.md

### PR DESCRIPTION
**Pull request by yomiadetutu1**
Updated to reflect Lattice

### CODE REVIEW

File: operations/workplace/people/performance-reviews-50/new-staff-performance-review.md [hunk 1]
```diff
@@ -9,13 +9,13 @@ The New Staff Performance Review is intended to provide align expectations and [
 ## Review Process
 
 * **Managers**
-* Log in to **Culture Amp &gt; Performance**. In your inbox, complete the request to select those who will make up the peer panel for the new staff member. For new hires who are people managers, you may also request upward feedback from their direct reports. We've already provided questions for you to use, so you don't need to worry about customizing them. Note that only FT Staff are included in the Culture Amp platform. If you wish to solicit feedback from PT Staff or from a community member, this must be done separately.
+* Log in to **Lattice &gt; Performance**. On your Lattice home page, complete the request to select those who will make up the peer panel for the new staff member. For new hires who are people managers, you may also request upward feedback from their direct reports. We've already provided questions for you to use, so you don't need to worry about customizing them. Note that only FT Staff are included in the Lattice platform. If you however wish to solicit feedback from PT Staff or from a community member, you may do so by entering their email address.
```
Comment (cwarnermm) [inline operations/workplace/people/performance-reviews-50/new-staff-performance-review.md]:
```suggestion
* Log in to **Lattice > Performance**. On your Lattice home page, complete the request to select those who will make up the peer panel for the new staff member. For new hires who are people managers, you may also request upward feedback from their direct reports. We've already provided questions for you to use, so you don't need to worry about customizing them. Note that only full-time Staff are included in the Lattice platform. If you however wish to solicit feedback from part-time Staff or from a community member, you may do so by entering their email address.
```

**Review by cwarnermm — DISMISSED** [review]
Feedback provided inline. Approving.

**Review by lynn915 — CHANGES_REQUESTED** [review]
Hey @yomiadetutu1 can you remove this, "For new hires who are people managers, you may also request upward feedback from their direct reports. We've already provided questions for you to use, so you don't need to worry about customizing them." 
Context: the upward feedback template is separate and will go out to eveyone - it doesn't have to be "requested".

File: operations/workplace/people/performance-reviews-50/new-staff-performance-review.md (hunk 1 above)
Comment (yomiadetutu1) [inline operations/workplace/people/performance-reviews-50/new-staff-performance-review.md]:
```suggestion
* Log in to **Lattice &gt; Performance**. On your Lattice home page, complete the request to select those who will make up the peer panel for the new staff member. Note that only FT Staff are included in the Lattice platform. If you however wish to solicit feedback from PT Staff or from a community member, you may do so by entering their email address.
```

**Review by lynn915 — APPROVED** [review]

**Review by cwarnermm — APPROVED** [review]

**Pull request merged**